cmd/config-reloader/Dockerfile

```diff
@@ -23,16 +23,29 @@ COPY go.sum go.sum
 COPY vendor* vendor
 COPY cmd cmd
 
-ENV GOEXPERIMENT=noboringcrypto
-ENV CGO_ENABLED=0
-ENV GOFIPS140=latest
+ENV GOEXPERIMENT=boringcrypto
+ENV CGO_ENABLED=1
+ENV GOFIPS140=off
 ENV GOOS=${TARGETOS}
 ENV GOARCH=${TARGETARCH}
-RUN go build \
-  -ldflags="-X github.com/prometheus/common/version.Version=$(cat charts/values.global.yaml | yq '.version' ) \
-  -X github.com/prometheus/common/version.BuildDate=$(date --iso-8601=seconds)" \
-  -o config-reloader \
-  cmd/config-reloader/*.go
+RUN if [ "${TARGETARCH}" = "arm64" ] && [ "${BUILDARCH}" != "arm64" ]; then \
+      apt install -y --no-install-recommends \
```

Comment:
question: what is this doing effectively?

We should already be supporting multi-arch builds, no?